docs/administrator/verify-artifacts.md

```diff
@@ -20,6 +20,8 @@ You need to install the following tools:
 
 Karmada introduced the `cosign` verification tool since release 1.7. For a list of published mirrors, see [karmada mirrors](https://hub.docker.com/u/karmada).
 
+> Note: image of helm chart introduced the `cosign` verification tool since release 1.8.
```

Comment:
```suggestion
> Note: The image of the helm chart has introduced the `cosign` verification tool since Karmada v1.8.
```

Reply:
> Karmada v1.8.

Thanks for review, The doc is really have style like `Karmada introduced the `cosign` verification tool since release 1.7. `,  This may not need to be modified for consistency?

Reply:
I can't understand why adding the notes under this chapter.
If we want to demonstrate how to verify Charts, we might need to organize the content as:
```
# Overview

## Images
## Charts    // new chapter
## Binaries
```
Note that this chapter is reserved when we setup the document at https://github.com/karmada-io/website/pull/449#pullrequestreview-1569063709.